packages/babel-plugin/src/utils/evaluate-expression/index.tsx

```diff
@@ -566,7 +405,11 @@ export const evaluateExpression = (
     // It's preferable to use the identifier than its result if it can't be statically evaluated.
     // E.g. say we got the result of an identifier `foo` as `bar()` -- its more preferable to return
     // `foo` instead of `bar()` for a single source of truth.
-    const babelEvaluatedNode = babelEvaluateExpression(value, updatedMeta, expression);
+    const babelEvaluatedNode = babelEvaluateExpression(
+      value as t.Expression,
```

Comment:
hmm got `value as t.Expression` back as part of a bad merge. Can probably remove the `as t.Expression`